docs/site/Repositories.md

```diff
@@ -133,199 +146,195 @@ export class AccountRepository extends DefaultCrudRepository {
 
-1. Implement the `CrudConnector` interface from `@loopback/repository` package. [Here is one way to do it](https://github.com/strongloop/loopback-next-example/blob/master/services/account-without-juggler/repositories/account/datasources/mysqlconn.ts)
-2. Implement the `DataSource` interface from `@loopback/repository`. To implement the `DataSource` interface, you must give it a name, supply your custom connector class created in the previous step, and instantiate it:
-    ```javascript
+1.  Implement the `CrudConnector` interface from `@loopback/repository` package.
+    [Here is one way to do it](https://github.com/strongloop/loopback-next-example/blob/master/services/account-without-juggler/repositories/account/datasources/mysqlconn.ts)
+
+2.  Implement the `DataSource` interface from `@loopback/repository`. To
+    implement the `DataSource` interface, you must give it a name, supply your
+    custom connector class created in the previous step, and instantiate it:
+
+    ```ts
     export class MySQLDs implements DataSource {
-      name: 'mysqlDs'
-      connector: MySqlConn
-      settings: Object
+      name: 'mysqlDs';
+      connector: MySqlConn;
+      settings: Object;
 
       constructor() {
-        this.settings = require('./mysql.json'); //connection configuration
+        this.settings = require('./mysql.json'); // connection configuration
         this.connector = new MySqlConn(this.settings);
       }
     }
     ```
-3. Extend `CrudRepositoryImpl` class from `@loopback/repository` and supply your custom DataSource and model to it:
 
-    ```javascript
-    import { CrudRepositoryImpl } from '@loopback/repository';
-    import { MySQLDs } from './datasources/mysqlds';
-    import { Account } from './models/Account';
+3.  Extend `CrudRepositoryImpl` class from `@loopback/repository` and supply
+    your custom DataSource and model to it:
+
+    ```ts
+    import {CrudRepositoryImpl} from '@loopback/repository';
+    import {MySQLDs} from './datasources/mysqlds';
+    import {Account} from './models/Account';
 
-    export class newRepository extends CrudRepositoryImpl<Account, string> {
+    export class NewRepository extends CrudRepositoryImpl<Account, string> {
       constructor() {
-        let ds = new MySQLDs();
+        const ds = new MySQLDs();
         super(ds, Account);
       }
     }
     ```
 
-You can override the functions it provides, which ultimately call on your connector's implementation of them, or write new ones.
+You can override the functions it provides, which ultimately call on your
+connector's implementation of them, or write new ones.
 
 ### Configure Controller
 
-The next step is to wire your new DataSource to your controller.
-This step is essentially the same as above, but can also be done as follows using DI:
+The next step is to wire your new DataSource to your controller. This step is
+essentially the same as above, but can also be done as follows using DI:
```

Comment:
i know this is not part of your change.   Should we using `Dependency Injection` instead of `DI` since it's the first occurrence of this term in this document?